### DISCUSSION TITLE
How to deploy custom reticulum to aws hubs cloud instance ?

### DISCUSSION THREAD

**Discussion by RathodDeven** [mozilla/hubs#5164] [Hubs Cloud Questions]
Is there any way to deploy custom reticulum to custom AWS hubs cloud instance ? If yes then how and if no then how can I edit  existing reticulum code ?

PS: I want to edit the default routing and change some api calls that are being called to reticulum

**Comment by pattersonbl2** [comment]
The process is a bit complex, but several users in our discord community have completed this. We don't have a how-to guide currently on this process. Our discord community has several channels for developers. https://discord.com/invite/dFJncWwHun
